src/testing/test-utils.ts

```diff
@@ -20,7 +20,7 @@ import ja from 'locale/ja.json';
 import _ from 'lodash';
 import path from 'path';
 import { getManifest as d2GetManifest } from '../app/bungie-api/destiny2-api';
-import profile from './data/profile-2023-02-28.json';
+import profile from './data/profile-2023-08-31.json';
```

Comment:
I updated the profile because my old test profile was old enough that it had a lot of invalid mods and such in it.